.github/workflows/build.yml

```diff
@@ -14,31 +14,26 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        php: [8.1, 8.2, 8.3, 8.4]
+        php: [ 8.1, 8.2, 8.3, 8.4 ]
         composer-flags: [ "" ]
         symfony-version: [ "" ]
         include:
-          -   php: 8.1
-              symfony-version: '^5.4'
-          -   php: 8.1
-              symfony-version: '^6.4'
-
+          - php: 8.1
+            symfony-version: '^5.4'
+          - php: 8.1
+            symfony-version: '^6.4'
     steps:
-      -   uses: actions/checkout@v4
-
-      -   name: Setup PHP
-          uses: shivammathur/setup-php@v2
-          with:
-            php-version: "${{ matrix.php }}"
-            ini-file: "development"
-            coverage: none
-
-      -   name: Update Symfony version
-          if: matrix.symfony-version != ''
-          run: composer require --no-update "symfony/symfony:${{ matrix.symfony-version }}"
-
-      -   name: Install dependencies
-          run: composer update ${{ matrix.composer-flags }}
-
-      -   name: Run tests (phpunit)
-          run: ./vendor/bin/phpunit
+      - uses: actions/checkout@v4
+      - name: Setup PHP
```

Comment:
Personally, I find it a lot easier to read / see / review YAML with a blank line between each block as before